src/lib/fastDownward/fastDownward.ts

```diff
@@ -245,7 +252,7 @@ export function search<LU extends LearningUnit>(
 				skillsNotFound.forEach(sk => remainSkills.push({ id: sk, repositoryId: "0", nestedSkills: [] }));
 				const tempLU = {
 					id: "-1",
-					requiredSkills: remainSkills,
+					requiredSkills: new And(remainSkills.map(skill => new Variable(skill))),
```

Comment:
I'm unsure. Is this realy the new intention, that we always create an `AND` expression at this point?

Reply:
I think that it does not matter which skill expression I use in this step ('Or', 'And', or 'Empty'), since I am just passing the missing skill (or the unreachable skill).